lite/examples/object_detection/android/app/src/main/java/org/tensorflow/lite/examples/detection/CameraActivity.java

```diff
@@ -17,278 +17,307 @@
 package org.tensorflow.lite.examples.detection;
 
 import android.Manifest;
```

Comment:
Imports should be sorted in alphabetic order.

Reply:
Noted!